guides/common/modules/proc_adding-a-vnc-connection-for-a-managed-host-on-proxmox.adoc

```diff
@@ -0,0 +1,43 @@
+[id="Adding_a_VNC_Connection_for_a_Managed_Host_on_Proxmox_{context}"]
+= Adding a VNC Connection for a Managed Host on Proxmox
+
+Use this procedure to add a VNC connection for a managed host on Proxmox.
+
+.Limitation
+* This procedure only works for hosts based on `qemu`.
+It does not work for LXC container on Proxmox.
+
+.Procedure
+. On your Proxmox, open the required port for VNC:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# firewall-cmd --add-port=_My_Port_/tcp --permanent
+# firewall-cmd --reload
+----
++
+The port is the sum of `5900` and the VMID of your managed host.
+You can view the VMID on the *VM* tab for your managed host in the {ProjectWebUI}.
+. On your Proxmox, enable VNC for your managed host in `/etc/pve/nodes/_My_Proxmox_Instance_/qemu-server/_My_VMID_.conf`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+-vnc 0.0.0.0:_My_VMID_,password=on
+----
++
+For more information, see https://pve.proxmox.com/wiki/VNC_Client_Access[VNC Client Access] in the Proxmox documentation.
+. Restart your managed host on Proxmox.
+. Optional: Verify that the required port for VNC is open:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# netstat -ltunp
+----
+. On your {ProjectServer}, allow websockify in SELinux:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# setsebool websockify_can_connect_all on
+----
+. If you use a {customssl} certificate, import the SSL certificate from {ProjectServer} into your browser.
```

Comment:
Should this section be Katello only?

Reply:
well, I guess, the browser certs are used in foreman-only, too?

Reply:
Changed to "katello only" because of https://github.com/theforeman/foreman-documentation/blob/master/guides/common/assembly_accessing-server.adoc?plain=1#L3